feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.html

```diff
@@ -35,7 +39,7 @@
   <ng-container *ngIf="isAttributeGroup(); else conflictGroup">
     <ng-container *ngIf="isNavigationToConflictEnabled(); else withoutLink">
       <a
-        class="cx-conflict-msg link cx-action-link"
```

Comment:
why did you remove 'cx-conflict-msg' class?

Reply:
See Slack